client/plots/sampleView.js

```diff
@@ -147,9 +147,8 @@ class SampleView {
 
 		if (config.samples?.length > 15) samples = config.samples.filter((s, i) => i < 15)
 		const q = appState.termdbConfig.queries
-		const hasPlots = q
-			? q.singleSampleGenomeQuantification || q.singleSampleMutation || (q.NIdata && showBrainImaging)
-			: false
+		const hasPlots = Object.keys(q).length > 0
```

Comment:
no it cannot detect by presence of any key in queries{}. it must only detect specific keys that correspond to sample-level plots

Reply:
this boolean is an optimization, I know the previous way was more precise, but I did not wanted to add a check for each new plot.

Reply:
it has to be like that. otherwise when there's only keys not applicable to single sample view, your flag is still true and wrong!

Reply:
Ok. I removed this variable and the logic related as it was not needed and confusing.